x-pack/solutions/observability/plugins/apm/server/routes/transactions/get_transaction/index.ts

```diff
@@ -97,7 +98,7 @@ export async function getTransaction({
       },
     },
     fields: [...requiredFields, ...optionalFields],
-    _source: [SPAN_LINKS, TRANSACTION_AGENT_MARKS],
+    _source: [SPAN_ID, SPAN_LINKS, TRANSACTION_MARKS_AGENT],
```

Comment:
Hey @cqbqdd11519 , I'm reviewing your PR, and I think the bug is not here, it's elsewhere. I'll get back to you once I have the confirmation.